### PR TITLE
 AKR-1906: Off-by-slash Path Travel Identified when performing request

### PR DESCRIPTION
**Pull request by juandesi**

### CODE REVIEW

File: src/main/java/org/mule/module/apikit/helpers/EventWrapper.java [hunk 1]
```diff
@@ -60,10 +60,13 @@ public EventWrapper doClientRedirect() {
...
   }
 
-
-
   public EventWrapper setPayload(Object payload, MediaType mediaType) {
     outputBuilder.message(MessageHelper.setPayload(inputEvent.getMessage(), payload, mediaType));
     return this;
   }
+
+  public EventWrapper setHttpStatus(String status) {
```
Comment (JuanAller) [inline src/main/java/org/mule/module/apikit/helpers/EventWrapper.java]: delete

Comment (JuanAller) [reply src/main/java/org/mule/module/apikit/helpers/EventWrapper.java]: validate file extensions

File: src/main/java/org/mule/module/apikit/api/RamlHandler.java [hunk 2]
```diff
@@ -235,7 +226,7 @@ public boolean isRequestingRamlV2(String listenerPath, String requestPath, Strin
       && requestPath.startsWith(resourcesFullPath);
   }
 
-  private String sanitarizeResourceRelativePath(String resourceRelativePath) {
+  private String uriPathToResourcePath(String resourceRelativePath) {
```
Comment (JuanAller) [inline src/main/java/org/mule/module/apikit/api/RamlHandler.java]: validate file extensions

Comment (juandesi) [reply src/main/java/org/mule/module/apikit/api/RamlHandler.java]: done

**Pull request merged**